presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -307,13 +307,10 @@ private void invokeNoMoreSplitsIfNecessary()
                         targetFilesystem,
                         partitionName,
```

Comment:
This field is no longer needed in the `InternalHiveSplitFactory`. Let's pass the `partitionName` to the `getBucketedSplits` directly, and remove this field from the `InternalHiveSplitFactory`